README.md

```diff
@@ -1 +1,57 @@
-# Edu-platform_Gateway
\ No newline at end of file
+# Edu-platform_Gateway
+
+This is the Api-Gateway project for the online learning platform. Its purpose is to handle all the requests and transfer them to the proper Web APIs.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Features](#features)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Introduction
+
+The Api-Gateway project serves as the central hub for managing and routing requests in the online learning platform. Its primary responsibility is to handle incoming requests and efficiently transfer them to the appropriate Web APIs within the system. This helps to streamline communication and ensure seamless interactions between different components of the online learning platform.
+
+## Getting Started
+
+Explain how to get the project up and running on a local machine.
+
+### Prerequisites
+
+The Project works on .Net 6 and is using next libraries:
+- Ocelot - the most importang NuGet Package, that handles all the routes and settings of gateway. 
+
+### Installation
+# Clone the repository
+git clone https://github.com/VitalikMatvieiev/Edu-platform-Gateway.git
+
+# Change directory
+cd your-project
+
+# Restore dependencies
+dotnet restore
+
+# Build the project
+dotnet build
+
+# Run the project
```

Comment:
Is it any other steps to run it from mac? Should I install additional packages to my machine?

Reply:
@Nazaziik